plugins/modelinspector/modelcontentdelegate.cpp

```diff
@@ -64,6 +64,12 @@ void ModelContentDelegate::paint(QPainter *painter, const QStyleOptionViewItem &
         // TODO also render selection for currently selected cells
     }
 
+    if (index.data(ModelContentProxyModel::EmptyNameRole).toBool()) {
+        option.palette.setColor(QPalette::Text, QPalette::Disabled);
+        option.text = tr("< unnamed at row:" + QString::number(index.row())
+                              + " column:" + QString::number(index.column()) + " >");
```

Comment:
This is neither translatable nor does it compile. Would need to be something like tr("<unnamed row %1 column %2>").arg(index.row()).arg(index.column())